### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by cmh2166**
- [x] #76 Complete & include dataflow diagram. 
- [ ] #76 Include Traject logic / override steps / workflow
- [x] #74 Won't solve, but add comment about in docs (data decision area)
- [x] #88 Add Steps for including records & mappings for local testing before inclusion
- [x] #98 Won't solve, but add comment about in docs (data decision area)
- [x] Include logic in data decisions area for current URL look-ups
- [x] #245 review + add to #285 (generic configs)
- [ ] #246 create docs on macros, when to add to, logic therein

Will not get through all of this, and there are other docs questions, but lining up for work.

### CODE REVIEW

**Comment by jcoyne** [comment]
what you've done so far is great. I wonder if we shouldn't remove the `[WIP]` and merge what is here and leave the remainder for other PRs?

**Comment by cmh2166** [comment]
Not getting to  #246 create docs on macros, when to add to, logic therein or  #76 Include Traject logic / override steps / workflow. leaving open for future work.

**Comment by anarchivist** [comment]
Going back to @jcoyne's point, I think we should merge this and ensure the remaining two tickets are up at the top of the backlog.

**Comment by cmh2166** [comment]
Yes @jcoyne + @anarchivist  i'm removing the WIP now.

I don't always see these comments so please also ping me in the slack channel.

File: docs/csv_data_transforms.md [hunk 1]
```diff
@@ -1,61 +0,0 @@
-# Information for DLME CSV Data Transforms
```
Comment (anarchivist) [inline docs/csv_data_transforms.md]: Did we delete this file for a reason?

File: docs/providers.md [hunk 2]
```diff
@@ -9,31 +9,33 @@ Currently, we have readers to accept metadata in the following formats:
...
 
-If certain information such as identifiers or links to digital objects or image thumbnails is unavailable from your metadata itself, we will likely ask for additional information about how to gather this data to build out quality records.
+If certain information such as identifiers, links to digital objects, or image thumbnails are unavailable from your metadata, we will likely ask for additional information about how to gather this data to build out quality records (e.g. with this identifier, and this URL structure, you can retrieve a thumbnail URL).
 
-More in-depth documentation is available below:
+## In-Depth Data Provider Documentation
 
-* [Application profile](application_profile.md)
-* [Metadata mappings](https://docs.google.com/spreadsheets/d/1Sp7uMHizVX7xN7xN9mm-vgEuESQBovXO-qenAo_TV-w/edit)
+* [Metadata ingested into the DLME needs to be mapped to the DLME Metadata Application Profile (MAP), or the available classes, fields, and definitions (including requirements) for objects described in DLME. You can read more details on the DLME MAP at this link](application_profile.md).
```
Comment (anarchivist) [inline docs/providers.md]: Can we link just part of these sentences? This fields a little unwieldly.

File: docs/csv_data_transforms.md (hunk 1 above)
Comment (cmh2166) [reply docs/csv_data_transforms.md]: It was added the data providers step by step bc all of the functions apply to all methods, not just CSV

File: docs/providers.md (hunk 2 above)
Comment (cmh2166) [reply docs/providers.md]: Sure

File: docs/csv_data_transforms.md (hunk 1 above)
Comment (cmh2166) [reply docs/csv_data_transforms.md]: it could be made into a generic functions page too, if that step by step is too long

Comment (anarchivist) [reply docs/csv_data_transforms.md]: I think it's OK - I overlooked that it was included in the add a data source doc.

**Comment by anarchivist** [comment]
tenor

**Pull request merged**